testing/py-influxdb/APKBUILD

```diff
@@ -43,4 +43,4 @@ _py() {
 	$python setup.py install --prefix=/usr --root="$subpkgdir" || return 1
```

Comment:
makes sense to remove this "return 1" as well - also why there's no `check()`?

Reply:
i removed the ` || return 1` and merge. It is a step forward.